readme.txt

```diff
@@ -116,6 +116,65 @@ You'll find the [FAQ on Yoast.com](https://yoast.com/wordpress/plugins/seo/faq/)
+	* Adds a primary category replacement variable: `%%primary_category%%`.
+	* Adds free screencasts to every settings page explaining that specific page.
+	* Introduces new help buttons in place of qtip, which makes these descriptions much more accessible.
+
+* Enhancements:
+	* Adds pinterest icon to the pinterest settings tab.
+	* Clarifies the text on the pinterest settings tab.
+	* Improves searchability of select inputs by using select2.
+	* Adds filters to customize sitemaps' <urlset>, props [Mark Walker](https://github.com/mnwalker).
+	* Uses wp_register_script and wp_register_style on init so other plugins can customize our assets.
+	* Changes text length for terms to 150 words.
+	* Removes analyses from the term analysis that weren't applicable to terms.
+	* Improves code architecture of sitemaps.
+	* Moves the onpage.org settings to the webmaster tab.
+	* Improves performance when importing or migrating posts, thanks [sun](https://github.com/sun).
+	* Adds caching to empty sitemaps.
+	* Adds parsing of shortcodes before recalculating all posts.
+	* Improves detection of static xml sitemaps
+	* Add target=_blank to external links inside the metabox help information, props [Borja Abad](https://github.com/mines).
+	* Clarifies the subpage of archives text.
+	* Removes noydir setting.
+	* Removes other tab from the import screen, these plugins have all been deprecated.
+	* Removes all settings to hide specific tags inside the head.
+	* Improves accessibility of add keyword modal.
+	* Improves accessibility of metabox.
+	* Switches all yoa.st links to be HTTPS.
+
+* Bugfixes:
+	* Fixes a bug where the breadcrumbs title field was hidden even though the theme supported breadcrumbs.
+	* Fixes a bug where underscores in like queries weren't correctly escaped, thanks [Konstantin Kovshenin](https://github.com/kovshenin) and [Damian Hodgkiss](https://github.com/damianhodgkiss)
+	* Fixes a bug where text inside a [caption] shortcode wouldn't be removed correctly in auto generated meta descriptions, props [Kevin Lisota](https://github.com/kevinlisota)
+	* Fixes a bug where a message to add headings to the text would only be shown if a keyword was set.
+	* Fixes a bug where a message to add links to the text would only be shown if a keyword was set.
+	* Fixes a compatibility issues with plugins that included mootools or prototypejs.
+	* Fixes a bug where the 404 page didn't correctly have a noindex and a nofollow set.
+	* Fixes a bug where internal taxonomies would be shown in the sitemap exlusion settings.
+	* Fixes a bug in the activation and deactivation where we would execute our code for every network, props [Felix Arntz](https://github.com/felixarntz).
+	* Fixes a bug where the primary category wasn't taken into account when calling `get_permalink` on the frontend.
+	* Fixes a compatibility issue with MultilingualPress, props [Thorsten Frommen](https://github.com/tfrommen).
+	* Fixes a compatibility issue with Easing Slider.
```

Comment:
Maybe add woocommerce "Variations and Swatches" here as well. The fix was build for Easing Slider, but Variations and Swatches had the exact same issue and is also solved by the same fix.